From c94a5eb0c14ddfb3a14061308001ac7dd2c42931 Mon Sep 17 00:00:00 2001
From: Diane Trout <diane@ghic.org>
Date: Sat, 24 Oct 2015 11:49:07 -0700
Subject: [PATCH 1/4] Relicense invalid license to LGPL 2.1+

Scott Kitterman noticed the license for this code was
"GPL 2.1+" which doesn't exist.

I (Diane Trout) tried to contact the author on a few occasions but
did not see a response.

I believe this code was created as a Google Summer of Code project and
so the following criteria which states that the student needs to choose
a license approved by the mentoring organization.

http://www.google-melange.com/gsoc/document/show/gsoc_program/google/gsoc2015/help_page?PageSpeed=noscript#2._What_licenses_do_I_have_to_choose

The most likely thing is that the original license block is a
typo and the student had first written GPL-3, but then changed it to
GPL 2.1, but forgot to add the "Lesser".

REVIEW: 125780
---
 otr-proxy/KTpProxy/main.cpp                      | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-config.cpp                | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-config.h                  | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-constants.h               | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-manager.cpp               | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-manager.h                 | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-message.cpp               | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-message.h                 | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-proxy-channel-adaptee.cpp | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-proxy-channel-adaptee.h   | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-proxy-channel.cpp         | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-proxy-channel.h           | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-session.cpp               | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-session.h                 | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-utils.cpp                 | 27 ++++++++++++------------
 otr-proxy/KTpProxy/otr-utils.h                   | 27 ++++++++++++------------
 otr-proxy/KTpProxy/pending-curry-operation.cpp   | 27 ++++++++++++------------
 otr-proxy/KTpProxy/pending-curry-operation.h     | 27 ++++++++++++------------
 otr-proxy/KTpProxy/proxy-observer.cpp            | 27 ++++++++++++------------
 otr-proxy/KTpProxy/proxy-observer.h              | 27 ++++++++++++------------
 otr-proxy/KTpProxy/proxy-service-adaptee.cpp     | 27 ++++++++++++------------
 otr-proxy/KTpProxy/proxy-service-adaptee.h       | 27 ++++++++++++------------
 otr-proxy/KTpProxy/proxy-service.cpp             | 27 ++++++++++++------------
 otr-proxy/KTpProxy/proxy-service.h               | 27 ++++++++++++------------
 otr-proxy/KTpProxy/types.h                       | 27 ++++++++++++------------
 otr-proxy/test/otr-test.cpp                      | 27 ++++++++++++------------
 26 files changed, 338 insertions(+), 364 deletions(-)

diff --git a/otr-proxy/KTpProxy/main.cpp b/otr-proxy/KTpProxy/main.cpp
index 87e1df6..feb703a 100644
--- a/otr-proxy/KTpProxy/main.cpp
+++ b/otr-proxy/KTpProxy/main.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "proxy-service.h"
diff --git a/otr-proxy/KTpProxy/otr-config.cpp b/otr-proxy/KTpProxy/otr-config.cpp
index 28c316c..fe1a8ee 100644
--- a/otr-proxy/KTpProxy/otr-config.cpp
+++ b/otr-proxy/KTpProxy/otr-config.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "otr-config.h"
diff --git a/otr-proxy/KTpProxy/otr-config.h b/otr-proxy/KTpProxy/otr-config.h
index ce42c00..2129ce0 100644
--- a/otr-proxy/KTpProxy/otr-config.h
+++ b/otr-proxy/KTpProxy/otr-config.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_OTR_CONFIG_HEADER
diff --git a/otr-proxy/KTpProxy/otr-constants.h b/otr-proxy/KTpProxy/otr-constants.h
index 08f1a14..28e1c39 100644
--- a/otr-proxy/KTpProxy/otr-constants.h
+++ b/otr-proxy/KTpProxy/otr-constants.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_OTR_CONSTANTS_HEADER
diff --git a/otr-proxy/KTpProxy/otr-manager.cpp b/otr-proxy/KTpProxy/otr-manager.cpp
index 6cbdfdf..ee65d64 100644
--- a/otr-proxy/KTpProxy/otr-manager.cpp
+++ b/otr-proxy/KTpProxy/otr-manager.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "otr-manager.h"
diff --git a/otr-proxy/KTpProxy/otr-manager.h b/otr-proxy/KTpProxy/otr-manager.h
index b8807aa..c7d62d9 100644
--- a/otr-proxy/KTpProxy/otr-manager.h
+++ b/otr-proxy/KTpProxy/otr-manager.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_OTR_MANAGER_HEADER
diff --git a/otr-proxy/KTpProxy/otr-message.cpp b/otr-proxy/KTpProxy/otr-message.cpp
index d475f40..9507f46 100644
--- a/otr-proxy/KTpProxy/otr-message.cpp
+++ b/otr-proxy/KTpProxy/otr-message.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "otr-message.h"
diff --git a/otr-proxy/KTpProxy/otr-message.h b/otr-proxy/KTpProxy/otr-message.h
index cd46f07..087b86a 100644
--- a/otr-proxy/KTpProxy/otr-message.h
+++ b/otr-proxy/KTpProxy/otr-message.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_OTR_VALUE_TYPES_HEADER
diff --git a/otr-proxy/KTpProxy/otr-proxy-channel-adaptee.cpp b/otr-proxy/KTpProxy/otr-proxy-channel-adaptee.cpp
index e30167f..05200f9 100644
--- a/otr-proxy/KTpProxy/otr-proxy-channel-adaptee.cpp
+++ b/otr-proxy/KTpProxy/otr-proxy-channel-adaptee.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "otr-proxy-channel-adaptee.h"
diff --git a/otr-proxy/KTpProxy/otr-proxy-channel-adaptee.h b/otr-proxy/KTpProxy/otr-proxy-channel-adaptee.h
index 78fd81b..615d5ba 100644
--- a/otr-proxy/KTpProxy/otr-proxy-channel-adaptee.h
+++ b/otr-proxy/KTpProxy/otr-proxy-channel-adaptee.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_OTR_PROXY_CHANNEL_ADAPTEE_HEADER
diff --git a/otr-proxy/KTpProxy/otr-proxy-channel.cpp b/otr-proxy/KTpProxy/otr-proxy-channel.cpp
index 025bebc..f3e6767 100644
--- a/otr-proxy/KTpProxy/otr-proxy-channel.cpp
+++ b/otr-proxy/KTpProxy/otr-proxy-channel.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "otr-proxy-channel.h"
diff --git a/otr-proxy/KTpProxy/otr-proxy-channel.h b/otr-proxy/KTpProxy/otr-proxy-channel.h
index daefe67..dccfc89 100644
--- a/otr-proxy/KTpProxy/otr-proxy-channel.h
+++ b/otr-proxy/KTpProxy/otr-proxy-channel.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_OTR_PROXY_CHANNEL_HEADER
diff --git a/otr-proxy/KTpProxy/otr-session.cpp b/otr-proxy/KTpProxy/otr-session.cpp
index 60c169f..1e1e0f7 100644
--- a/otr-proxy/KTpProxy/otr-session.cpp
+++ b/otr-proxy/KTpProxy/otr-session.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "otr-session.h"
diff --git a/otr-proxy/KTpProxy/otr-session.h b/otr-proxy/KTpProxy/otr-session.h
index de1b381..fb9aa6e 100644
--- a/otr-proxy/KTpProxy/otr-session.h
+++ b/otr-proxy/KTpProxy/otr-session.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_OTR_SESSION_HEADER
diff --git a/otr-proxy/KTpProxy/otr-utils.cpp b/otr-proxy/KTpProxy/otr-utils.cpp
index 4b50622..a88aadd 100644
--- a/otr-proxy/KTpProxy/otr-utils.cpp
+++ b/otr-proxy/KTpProxy/otr-utils.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "otr-utils.h"
diff --git a/otr-proxy/KTpProxy/otr-utils.h b/otr-proxy/KTpProxy/otr-utils.h
index fd80d14..d014c86 100644
--- a/otr-proxy/KTpProxy/otr-utils.h
+++ b/otr-proxy/KTpProxy/otr-utils.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_OTR_UTILS_HEADER
diff --git a/otr-proxy/KTpProxy/pending-curry-operation.cpp b/otr-proxy/KTpProxy/pending-curry-operation.cpp
index f5eee49..87d52ab 100644
--- a/otr-proxy/KTpProxy/pending-curry-operation.cpp
+++ b/otr-proxy/KTpProxy/pending-curry-operation.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "pending-curry-operation.h"
diff --git a/otr-proxy/KTpProxy/pending-curry-operation.h b/otr-proxy/KTpProxy/pending-curry-operation.h
index 80e3fcc..25a8f40 100644
--- a/otr-proxy/KTpProxy/pending-curry-operation.h
+++ b/otr-proxy/KTpProxy/pending-curry-operation.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_PENDING_CURRY_OPERATION_HEADER
diff --git a/otr-proxy/KTpProxy/proxy-observer.cpp b/otr-proxy/KTpProxy/proxy-observer.cpp
index f8c715d..13c4242 100644
--- a/otr-proxy/KTpProxy/proxy-observer.cpp
+++ b/otr-proxy/KTpProxy/proxy-observer.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "proxy-observer.h"
diff --git a/otr-proxy/KTpProxy/proxy-observer.h b/otr-proxy/KTpProxy/proxy-observer.h
index 5b37e02..23441b6 100644
--- a/otr-proxy/KTpProxy/proxy-observer.h
+++ b/otr-proxy/KTpProxy/proxy-observer.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_OBSERVER_HEADER
diff --git a/otr-proxy/KTpProxy/proxy-service-adaptee.cpp b/otr-proxy/KTpProxy/proxy-service-adaptee.cpp
index 509349d..ac0f132 100644
--- a/otr-proxy/KTpProxy/proxy-service-adaptee.cpp
+++ b/otr-proxy/KTpProxy/proxy-service-adaptee.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "proxy-service-adaptee.h"
diff --git a/otr-proxy/KTpProxy/proxy-service-adaptee.h b/otr-proxy/KTpProxy/proxy-service-adaptee.h
index 25b5e1e..af694c4 100644
--- a/otr-proxy/KTpProxy/proxy-service-adaptee.h
+++ b/otr-proxy/KTpProxy/proxy-service-adaptee.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_PROXY_SERVICE_ADAPTEE_HEADER
diff --git a/otr-proxy/KTpProxy/proxy-service.cpp b/otr-proxy/KTpProxy/proxy-service.cpp
index b5e68b1..ba6bcfe 100644
--- a/otr-proxy/KTpProxy/proxy-service.cpp
+++ b/otr-proxy/KTpProxy/proxy-service.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "proxy-service.h"
diff --git a/otr-proxy/KTpProxy/proxy-service.h b/otr-proxy/KTpProxy/proxy-service.h
index 31f5e58..79f3bd2 100644
--- a/otr-proxy/KTpProxy/proxy-service.h
+++ b/otr-proxy/KTpProxy/proxy-service.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_PROXY_SERVICE_HEADER
diff --git a/otr-proxy/KTpProxy/types.h b/otr-proxy/KTpProxy/types.h
index deee1b6..6591228 100644
--- a/otr-proxy/KTpProxy/types.h
+++ b/otr-proxy/KTpProxy/types.h
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #ifndef KTP_PROXY_TYPES_HEADER
diff --git a/otr-proxy/test/otr-test.cpp b/otr-proxy/test/otr-test.cpp
index 5979e10..48baca6 100644
--- a/otr-proxy/test/otr-test.cpp
+++ b/otr-proxy/test/otr-test.cpp
@@ -1,20 +1,19 @@
 /***************************************************************************
  *   Copyright (C) 2014 by Marcin Ziemiński <zieminn@gmail.com>            *
  *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2.1 of the License, or   *
- *   (at your option) any later version.                                   *
- *                                                                         *
- *   This program is distributed in the hope that it will be useful,       *
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
- *   GNU General Public License for more details.                          *
- *                                                                         *
- *   You should have received a copy of the GNU General Public License     *
- *   along with this program; if not, write to the                         *
- *   Free Software Foundation, Inc.,                                       *
- *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
+ * This library is free software; you can redistribute it and/or           *
+ * modify it under the terms of the GNU Lesser General Public		   *
+ * License as published by the Free Software Foundation; either		   *
+ * version 2.1 of the License, or (at your option) any later version.	   *
+ * 									   *
+ * This library is distributed in the hope that it will be useful,	   *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of	   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU	   *
+ * Lesser General Public License for more details.			   *
+ * 									   *
+ * You should have received a copy of the GNU Lesser General Public	   *
+ * License along with this library; if not, write to the Free Software	   *
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA*
  ***************************************************************************/
 
 #include "lib/test-config.h"
-- 
2.6.2

